src/Person.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <string>
+
+
+class Person
+{
+public:
```

Comment:
everything is public - not good. The person itself should have proper constructor to initialize all values.